nibabel/parrec.py

```diff
@@ -329,6 +329,21 @@ def _process_image_lines(image_lines, version):
     return image_defs
 
 
+def _direction_numbers(bvecs):
```

Comment:
Maybe put in a test for this helper function?  You could copy https://github.com/yarikoptic/nitest-balls1/blob/master/PARREC/DTI.PAR into the `tests/data` directory to get some sample bvecs.